osbs/api.py

```diff
@@ -241,6 +241,17 @@ def _get_running_builds_for_build_config(self, build_config_id):
                 running.append(br)
         return running
 
+    def _get_not_cancelled_builds_for_koji_task(self, koji_task_id):
+        all_builds_for_task = self.os.list_builds(koji_task_id=koji_task_id).json()['items']
+        not_cancelled = []
+
+        for b in all_builds_for_task:
```

Comment:
Looks like there's a shorter version:
~~~~
responses = (BuildResponse(b) for b in all_builds_for_task)
not_cancelled = [br for br in responses if not br.is_cancelled()]
~~~~

Reply:
I don't think substituting the for-loop with two list comprehensions makes things clearer in this case.